src/components/ComponentPages/HistoryContainer/Collapse/index.tsx

```diff
@@ -245,11 +245,11 @@ function HistoryCollapse({
                           Close
                         </Button>,
                       ]}
+                      onCancel={() => setModal1Open(false)}
                     >
                       <p>
-                        To object to this change, you should be a direct
-                        supporter of the topic/camp and should have supported it
-                        before the change was submitted.
+                        Only direct supporters at the time this change was
```

Comment:
I would suggest, move it to common message file

Reply:
Done